dev/ci/user-overlays/maximedenes-ssr-merge.sh

```diff
@@ -0,0 +1,4 @@
+if [ $TRAVIS_PULL_REQUEST == "669" ] || [ $TRAVIS_BRANCH == "ssr-merge" ]; then
+    mathcomp_CI_BRANCH=ssr-merge
+    mathcomp_CI_GITURL=https://github.com/maximedenes/math-comp.git
+fi
```

Comment:
I changed the recommendation to include the PR number. I don't think using the `:` separator is a good idea though because the file name wouldn't be OK in all filesystems. I kept the `.sh` instead of `.overlay` because editors detect this for syntax highlighting (and these files should be in a `user-overlays/` directory anyways.